packages/tasks/src/model-libraries.ts

```diff
@@ -185,6 +185,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.mlx,
 		filter: true,
 	},
+	musicgen: {
```

Comment:
I think it's weird the library is `audiocraft` but the filter is `musicgen`. I would use `audiocraft` everywhere, that will scale better + we can then use this for other architectures as well (audiogen, encodec, etc).

Reply:
But there is no filter right? We explicitly mark `filter: false` - audiocraft can have quite a lot of different APIs for different models - magnet + audiogen.